testing/dart/image_filter_test.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+import 'dart:ui';
+
+import 'package:test/test.dart';
+
+const Color green = Color(0xFF00AA00);
+
+const int greenCenterBlurred = 0x1C001300;
+const int greenSideBlurred = 0x15000E00;
+const int greenCornerBlurred = 0x10000A00;
+
+const int greenCenterScaled = 0xFF00AA00;
+const int greenSideScaled = 0x80005500;
+const int greenCornerScaled = 0x40002B00;
+
+// This file requires the ImageFilter blur and matrix to both be supported
+// and have working ==, hashcode and toString() implementations.  The web
+// versions of these classes are not quite there yet, but will be soon.
+@TestOn('!chrome')
```

Comment:
You shouldn't need this as we do not run these tests on the Web. All web tests are under `lib/web_ui/test`.

Reply:
I thought I remembered the pre-submit checks trying to run these on web before, but I may be mistaken there. I'll remove it and try again.